build-system/server.js

```diff
@@ -79,6 +79,8 @@ app.use('/form/echo-json/post', function(req, res) {
     if (fields['email'] == 'already@subscribed.com') {
       res.statusCode = 500;
     }
+    res.setHeader('AMP-Access-Control-Allow-Source-Origin',
+        req.protocol + '://' + req.headers.host);
```

Comment:
Apparently that's not true in the [the node server env](http://expressjs.com/en/api.html#req.protocol).